test/vc-data-model-1.0/input/example-5-bad-proof-missing-type.jsonld

```diff
@@ -16,8 +16,9 @@
   },
   "proof": {
     "created": "2017-06-18T21:19:10Z",
-    "creator": "https://example.com/jdoe/keys/1",
-    "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
-    "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+MCRVpjOboDoe4SxxKjkCOvKiCHGDvc4krqi6Z1n0UfqzxGfmatCuFibcC1wpsPRdW+gGsutPTLzvueMWmFhwYmfIFpbBu95t501+rSLHIEuujM/+PXr9Cky6Ed+W3JT24="
+    "proofPurpose": "authentication",
```

Comment:
```suggestion
    "proofPurpose": "assertionMethod",
```

Reply:
This is on a VC, so the purpose of the proof is as a method of assertion, it is not for authentication.